modules/ROOT/pages/import-export-an-organization.adoc

```diff
@@ -1,7 +1,7 @@
 = Install/export an organization
-:description: This page explains what a user with the _Administrator_ profile in Bonita Portal or in the Bonita Administrator Application can see and do about the Organization of Bonita users (an .xml file).
+:description: This page explains what a user with in the Bonita Administrator Application can see and do about the Organization of Bonita users (an .xml file).
```

Comment:
his page explains what a user with in the Bonita Administrator Application can  => his page explains what a user with _Administrator_ profile in the Bonita Administrator Application can

Reply:
I wanted to remove the Administrator profile since it's redundant (only that profile has access to Bonita Administrator Application)

Reply:
How about "This page explains what a user can see and do in the Bonita Administrator Application about the Organization of Bonita users (an .xml file)."

Reply:
ok for the proposal.

Reply:
I added it